packages/embeds/embed-react/floating.html

```diff
@@ -0,0 +1,10 @@
+<html>
+  <head>
```

Comment:
Made a separate test page to test floating popup with embed-react. Earlier it was tested along with inline and it had side effects of inline usage.